locals.tf

```diff
@@ -63,8 +63,6 @@ locals {
     tags                          = local.tags
   }
 
-  ecr_image_repo_url = "${local.context.aws_caller_identity_account_id}.dkr.ecr.${local.context.aws_region_name}.amazonaws.com"
```

Comment:
Curious, is this because we don't use it anywhere?

Reply:
Yes it's not used anywhere. This was reported by the tflint